asdf_astropy/converters/table/tests/test_table.py

```diff
@@ -0,0 +1,279 @@
+import asdf
+from asdf.tests.helpers import yaml_to_asdf
+from astropy.coordinates import SkyCoord, EarthLocation
+from astropy.table import Table, NdarrayMixin, QTable, Column, MaskedColumn
+from astropy.time import Time, TimeDelta
+import astropy.units as u
+import numpy as np
+from numpy.testing import assert_array_equal
+import pytest
+
+from asdf_astropy.testing.helpers import (
+    assert_earth_location_equal,
+    assert_sky_coord_equal,
+    assert_time_equal,
+    assert_time_delta_equal,
+)
+
+
+def assert_description_equal(a, b):
+    if a in ("", None) and b in ("", None):
+        return
+
+    assert a == b
```

Comment:
Since this is a pytest assertion helper, you may want to include

```
__tracebackhide__ = True
```

in this function (and the assertion helpers below) so that when the assertion fails, it reports and rewrites the failure instead of regurgitating the code from the assertion helper.  This makes debugging failing tests much cleaner.

Reply:
I had `__tracebackhide__` originally, but since these helpers include multiple assertions, I found that I was missing useful detail on which assertion failed.